lib/translator/inquiry.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_INQUIRY_H
+#define LIB_TRANSLATOR_INQUIRY_H
+
+#include <cstdio>
+
+#include "absl/types/span.h"
+#include "common.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
+
+namespace translator {
+void translate(absl::Span<const uint8_t>);
+StatusCode RawToScsiCommand(absl::Span<const uint8_t>,
+                            scsi_defs::InquiryCommand &);
+
+scsi_defs::InquiryData BuildStandardInquiry();
```

Comment:
In general, it is good practice to add comments describing all methods and functions available in a header file. Sometimes the function name and signature is enough, but here it would be good to document what the caller can expect of different functions.

For example, it is not clear what  `BuildStandardInquiry` will return. Is it an empty InquiryData object? In this case, it returns a InquiryData struct with some default values set, so a high level description of this would be good.